ci/build/build-docker-images.py

```diff
@@ -881,5 +885,60 @@ def main(
         # push_readmes(build_type is MERGE)
 
 
+def fix_docker_images(
+    image: str = "ray-ml",
+    version: str = "nightly",
+    repo: str = DOCKER_HUB_REPO,
+):
+    tags = create_image_tags(
+        image_name=image,
+        py_versions=list(PY_MATRIX.keys()),
+        image_types=list(BASE_IMAGES.keys()),
+        specific_tag=None,  # Set to `latest` for latest image fixes
+        version=version,
+        suffix=None,
+    )
+    print(dict(tags))
+
+    # Pull images we want to rebuild
+    for base_tag in tags:
+        base_image = f"{repo}/{image}:{base_tag}"
+
+        print(f"docker pull {base_image}")
```

Comment:
what is this for loop supposed to be doing? Do we actually need to pull here?

Reply:
Yes, because we retag below and this only works if the image is available.

It's also good in order to find out if any images are not found.

Reply:
Sorry I mean, this is just a print statement. Do we actually need to run the pull command? Same thing for other print statements in this function.